app/controllers/organisations_controller.rb

```diff
@@ -16,6 +16,7 @@ def index
 
     all_organisations = Organisation.order(:name)
     @pagy, @organisations = pagy(filtered_collection(all_organisations.visible, search_term))
+    @merge_requests = MergeRequest.visible.order(:new_organisation_name)
```

Comment:
order by the absorbing_organisation.name here?

Reply:
Yes missed that